### PR TITLE
7917 Update customers logos on Homepage

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-7917

### CODE REVIEW

**Comment by MaximKraev** [comment]
please do not forget to put original images to codio/assests

**Comment by denpira** [comment]
Original images added to codio/assets

File: source/index.html.erb [hunk 1]
```diff
@@ -179,10 +179,14 @@ description: Codio is the platform used by educators to teach computer science.
         <%= image_tag 'assets/index/logos/aga_khan.png' %>
         <%= image_tag 'assets/index/logos/duck_duck.png' %>
         <%= image_tag 'assets/index/logos/durham.png' %>
-        <%= image_tag 'assets/index/logos/kent_state.png' %>
+        <!--<%= image_tag 'assets/index/logos/kent_state.png' %>-->
         <%= image_tag 'assets/index/logos/ocr.png' %>
         <%= image_tag 'assets/index/logos/partners-cas.png' %>
         <%= image_tag 'assets/index/logos/partners-naace.png' %>
+        <%= image_tag 'assets/index/logos/csta-logo.png' %>
```
Comment (MaximKraev) [inline source/index.html.erb]: I think we need an alt tag for every image with the list of partners names in it, for SEO purpose.

Comment (denpira) [inline source/index.html.erb]: Done

**Comment by MaximKraev** [comment]
:+1:

**Pull request merged**